enforcer-rules/src/site/apt/requireSameVersions.apt.vm

```diff
@@ -37,8 +37,10 @@
    * <<reportPlugins>> - an optional list of report plugin patterns
    
    * <<plugins>> - an optional list of both build and report plugin patterns
-    
+   
    * <<uniqueVersions>> - if SNAPSHOTs should be compared by their timestamped version or not. Default: false
+   
+   * <<sameModuleVersions>> - if set to true enforces that the current module has the same version as the top level aggregator project. Default: false. Supported since version 3.6.0

```

Comment:
We don't have a `since` sentence for other options even for rule itself.
So maybe we should be consistent.

Reply:
Not having it is IMHO worse, adding it to the other parameters is out of scope. Do you want me to remove that hint?

Reply:
hint can stay - no problem for me